package.json

```diff
@@ -59,7 +59,6 @@
     "react-dropzone": "^3.6.0",
     "react-ga": "1.4.1",
     "react-hot-loader": "1.3.0",
-    "react-loader": "2.4.0",
```

Comment:
can you make sure `npm-shrinkwrap.json` is updated to account for this change? if you do a fresh install with the `package.json` and `npm-shrinkwrap.json` from `master` and then do `npm uninstall --save react-loader` that should do the trick.